packages/grid/_modules_/grid/hooks/features/useColumnResize.tsx

```diff
@@ -124,5 +247,11 @@ export const useColumnResize = (columnsRef: React.RefObject<HTMLDivElement>, api
     };
   }, [stopListening]);
 
-  return React.useMemo(() => ({ onMouseDown: handleMouseDown }), [handleMouseDown]);
+  return React.useMemo(
+    () => ({
+      onMouseDown: handleMouseDown,
+      onTouchStart: handleTouchStart,
```

Comment:
Should be a ref to allow no passive event.

Reply:
This one needs to be pass around, and todo that we'll do property drilling. How about we do the same as the column reordering so we avoid that and we do the same pattern in the whole component.

Reply:
From what I understand, property drilling will be required for both the customization API, the unstyled components and the composability of the plugin. What's the advantage of avoiding it?

Reply:
@DanailH See for more details: https://github.com/facebook/react/pull/19654 why we need a ref.

Reply:
mm I guess that is because of Safari. Ok I'll see how to delegate that event because I don't want to attach it to each of the separators.

Reply:
We support Safari, starting at 12.1. touch-action none is supported after 13: https://en.wikipedia.org/wiki/Safari_version_history#Safari_12. We could decide to degrade the UX on 12.1 if it's significantly simpler. In such a case, we would drop `doesSupportTouchActionNone`, it's useless.

Reply:
I did some testing, It's quite bad that you can scroll and resize at the same time. For the slider, it wasn't so bad but the grid is basically unusable if there is a horizontal scroll and you try to decrease the size of a cell => the outcome is that you can because you scroll and resize left at the same time which breaks the resize logic due to coordinates change.

I've updated the event listener.